server/datastore/mysql/schema.sql

```diff
@@ -193,7 +193,6 @@ CREATE TABLE `host_users` (
   `created_at` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
   `removed_at` timestamp NULL DEFAULT NULL,
   `user_type` varchar(255) DEFAULT NULL,
-  `shell` varchar(255) DEFAULT '',
```

Comment:
We should revert these. Something is messing up the final schema, I'll see if I can get to the bottom of this. At first it only happened to me, but now it seems it's happening to everyone.

Reply:
What steps should I follow to revert properly?

Reply:
For now `git checkout main -- server/datastore/mysql/schema.sql` should work, I'll see if I can understand the underlying issue.

Reply:
Well, of course I can't reproduce this issue anymore... if you go to main on a clean branch, and you do `make dump-test-schema`, does it reproduce for you? As in, those columns are removed after the dump.

Reply:
It doesn't seem to reproduce on main for me. The columns are still there after the dump.

Reply:
@chiiph just wanted to confirm that I should still re-commit the schema here with the change to the `app_config_json` adding the default zendesk value?

Reply:
Not sure I understand the question. The main issue is that this is removing columns that should remain. If you regenerate the schema with `make dump-test-schema` it should fix that. If you checkout the one in main, you will need to add whatever changes you had by hand while we figure out what's wrong with the dump.